templates/medical/blood_donation.txt

```diff
@@ -0,0 +1,46 @@
+[center][b]Blood Donation Informed Consent Form[/b][/center]
+[hr]
+Have you donated blood before? [field] [small](yes/no)[/small][br]
+[hr]
+[u]For Repeat Donors[/u][br]
+Did you encounter any problems in your last donation? [field]
+[small](No problems / fainting / bruise / difficulties finding vein / other)[/small][br]
+[hr]
+Blood type: [field][br]
+Date of Birth: [field] Sex: [field] Height: [field] Weight: [field][i]kg[/i][br]
+If more than 60-70 years old, Do you have medical certificate? [field] [small](yes/no)[/small][br]
+Occupation: [field][br]
+Name: [i]Mr. / Ms. / Mrs.[/i] [field] Signature: [field][br]
+[hr]
+[hr]
+[center][b]For your own safety and the safety of the patient who will receive your blood, please answer the following questions, if applicable, to the best of your knowledge:[/b][/center]
+[hr]
+[u]Obstetric History[small](if applicable)[/small][/u][br]
```

Comment:
If we're leaving "Category 2" and "Category 3," may as well label this one "Category 1"
```suggestion
Category 1: [u]Obstetric History[small](if applicable)[/small][/u][br]
```